recipes-go/3mdeb-rtectrl/3mdeb-rtectrl/flash.sh

```diff
@@ -0,0 +1,347 @@
+#!/bin/sh
+
+errorExit() {
+  errorMessage="$1"
+  echo "$errorMessage"
+  exit 1
+}
+
+errorCheck() {
+  errorCode=$?
+  errorMessage="$1"
+  [ "$errorCode" -ne 0 ] && errorExit "$errorMessage : ($errorCode)"
+}
+
+usage() {
+  echo "${0} [COMMAND] [OPTIONS]"
+  echo "[COMMAND]:"
+  echo "    help - show this help"
+  echo "    probe - probe flash"
+  echo "    read [FILE] - read flash to FILE"
+  echo "    write [FILE] - write flash from FILE"
+  echo "    wp-list - check WP list"
+  echo "    wp-status - check WP status"
+  echo "    wp-enable - enable WP"
+  echo "    wp-disable - disable WP"
+  echo "    wp-high - enable hardware protection of status registers"
+  echo "    wp-low -disable hardware protection of status registers"
+  echo "[OPTIONS]:"
+  echo "    -V --voltage [VOLTAGE] - specify SPI voltage to be used: 1.8V or 3.3V"
+  echo "    --flashrom-params \"[PARAMS]\" - specify additional flashrom params surrounded"
+  echo "    with double quotes"
+  echo "    -v --verbose - verbose mode"
+  echo "    -d --dry-run - run skript without executing commands"
+  echo "Examples:"
+  echo "${0} probe"
+  echo "${0} read flash.rom"
+  echo "${0} write flash.rom"
+  exit 0
+}
+
+SPI_DEV="/dev/spidev1.0"
+SPI_SPEED="16000"
+FLASHROM_PROGRAMMER="linux_spi:dev=$SPI_DEV,spispeed=$SPI_SPEED"
+
+WP_RANGE="0,0x00800000"
+
```

Comment:
This script [had](https://github.com/3mdeb/RteCtrl/blob/fa69b610777d85113e005451122fff2a1011b54f/scripts/flash.sh#L32) the following unused variables here:
```bash
CHIP_THINKPAD_SPI1="MX25L6406E/MX25L6408E"
CHIP_THINKPAD_SPI2="MX25L3206E/MX25L3208E"
CHIP_PROTECTLI_FW6="MX25L6436E/MX25L6445E/MX25L6465E"
# FLASHROM_CHIP="-c $CHIP_PROTECTLI_FW6"
```

I wonder, should I utilize them here or it would be ok for user to provide these via `--flashrom-params`?

Reply:
These unused variables can be dropped. If more extra parameters to flashrom are to be added, user can do so via specific parameter